demo/client/src/ml-models/deploy-model.ts

```diff
@@ -35,12 +35,11 @@ export class ModelDeployer {
 	async deployNaiveBayes(model: NaiveBayesModel, options: any): Promise<Contract> {
 		const { account, toFloat,
 			notify, dismissNotification,
-			saveTransactionHash, saveAddress,
+			saveTransactionHash, saveAddress, initialFeatureChunkSize = 150, featureChunkSize = 350
```

Comment:
To keep the names consistent, would you use `initialChunkSize` and `chunkSize` here?

Reply:
sure will do that